knative-operator/Gopkg.toml

```diff
@@ -2,10 +2,6 @@
   name = "github.com/go-openapi/spec"
   branch = "master"
 
-[[override]]
```

Comment:
Why is this override ineffective ?

Reply:
Because the thing wasn't vendored at all.